### ISSUE TITLE
 Accessibility issue: insufficient label on attribution box close button

### ISSUE THREAD

**Issue by scoutb-cogapp** [UniversalViewer/universalviewer#1075]
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
close button for attribution box

### Issue description
The accessible label for the close button for the attribution box is "close". It is hard for a screenreader user to assess what this button is meant to close. 
Even having the text above and below the button read out by the screenreader will not make it 100 percent obvious what the button closes (especially as the button appears to be positioned between the heading of the box and its content).

### Steps to reproduce
To observe what a screenreader user experiences:
1. activate a screenreader
2. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
3. navigate to the attribution box by tabbing and listen to what is announced
4. attempt to find out more about the close button by further navigating using the keyboard and screenreader

### Expected behaviour
Button should be announced in a way that a non-sighted user would understand its context and purpose.

### Possible fix
Change accessible label to something along the lines of "close attribution box"

### WCAG criterion
2.4.4 Link Purpose (In Context) (Level A)

### Related code

```html
<button type="button" class="close" aria-label="Close"> <span aria-hidden="true">×</span> </button>
```

**Comment by LlGC-mag** [comment]
I'll have a crack at this one

**Comment by LlGC-mag** [comment]
Code should be ready for this one, I just need translations for "Close attribution panel" in French, Polish, and Swedish before committing

**Comment by demiankatz** [comment]
@LlGC-mag, do you have a PR for this one open yet, or are you waiting for the translations first?

**Comment by LlGC-mag** [comment]
@demiankatz James kindly got in touch with his friendly Swede so I'm just waiting for French and Polish. 

Not sure what's preferred with UV translations - would you normally submit the PR / merge and then add translations later or wait for the translations so it was complete before merging?

**Comment by demiankatz** [comment]
> Not sure what's preferred with UV translations - would you normally submit the PR / merge and add translations later or wait for the translations so it's complete before merging?

Yes, I'd suggest opening the PR right away but putting it in "draft" mode so it doesn't get merged prematurely. That way, if any code review is needed, that can happen in parallel with the translation work.
